lib/posix/pthread.c

```diff
@@ -718,7 +720,7 @@ static int posix_thread_pool_init(void)
 
 
 	for (i = 0; i < CONFIG_MAX_PTHREAD_COUNT; ++i) {
-		posix_thread_pool[i].state = PTHREAD_EXITED;
+		posix_thread_pool[i].state = PTHREAD_TERMINATED;
```

Comment:
as above, if `PTHREAD_TERMINATED` were to be assigned the value 0, then this wouldn't be necessary.

Reply:
> as above, if `PTHREAD_TERMINATED` were to be assigned the value 0, then this wouldn't be necessary.
> 

IIRC it was changed so that there was agreement between detached state and joinable among the various libcs

Reply:
Ah, that's slightly annoying, but understandable. Probably better to have this initialized here than stick the thread pool in data with a static initializer.